frontend/src/app/commonComponents/ChangeUser.tsx

```diff
@@ -4,12 +4,14 @@ const ChangeUser = () => {
   }
 
   const getLink = (email: string, name: string) => (
-    <a
-      className="ghost-user-link"
-      href={`/#access_token=SR-DEMO-LOGIN%20${email}`}
-    >
-      Login as {name}
-    </a>
+    <li>
```

Comment:
⚠️ caught by axe-jest, cannot have `<li>` element within `<ul>` parent element

Reply:
just to clarify, did you mean that we can't have a `<a>` element within a `<ul>`? wanted to make sure I understood for future reference!

Reply:
ooo sorry I missed this comment. I think there were only three elements that could be direct children of the `<ul>` element... let me try to see if I can get that error again.

Reply:
@fzhao99 it was this error! https://dequeuniversity.com/rules/axe/4.7/list?application=AxeChrome